### PR TITLE
Edits and updates made to README.rst

### PR DESCRIPTION
**Pull request by aefitts**
Added installation and contribution instructions as well as links to newly added vignette gallery to README.rst. Updated features and contributors. @nkorinek @lwasser please let me know if we should add any additional information, thank you!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,19 +19,152 @@ StreamStats
 
 Python package for interfacing with the USGS StreamStats API.
 
+- Free software: MIT license
+- Documentation: https://streamstats-python.readthedocs.io/en/latest/
+- GitHub: https://github.com/earthlab/streamstats
```
Comment (nkorinek) [inline README.rst:24]: Probably don't need a link to the GitHub page since they are already here!

File: README.rst [hunk 2]
```diff
@@ -20,19 +19,152 @@ StreamStats
...
+for applied examples of using StreamStats.
+
+
+Installation
+~~~~~~~~~~~~~
+Stable release
+--------------
+To install StreamStats via ``pip`` use:
+
+.. code-block:: python
```
Comment (nkorinek) [inline README.rst:46]: I don't think it matters to much, but I believe this code block should be console, not python.

File: README.rst [hunk 3]
```diff
@@ -20,19 +19,152 @@ StreamStats
...
+    $ conda install -c conda-forge streamstats
+
+
+From sources
+------------
+The sources for StreamStats can be downloaded from the `GitHub repository <https://github.com/earthlab/streamstats>`_ .
+
+You can either clone the public repository:
+
+.. code-block:: python
```
Comment (nkorinek) [inline README.rst:70]: Same for all these code blocks that say python. Should all be console!

**Review by nkorinek — CHANGES_REQUESTED** [review]
Hey @aefitts great job! Some VERY minor comments and we should be good to merge this.

**Comment by aefitts** [comment]
Thank you for the feedback @nkorinek the revisions have been made and we should be all set to merge!

**Pull request merged**